roles/agnosticv/templates/resource_provider.yml.j2

```diff
@@ -80,7 +80,7 @@ spec:
 {% set _update_filters_empty = true %}
 
 {# Allow changing start schedule if not explicitly disabled #}
-{% if 'start' not in _entry_points or _entry_points.start | default('', true) != '' %}
+{% if not _actions.start.disabled | default(false) | bool and ('start' not in _entry_points or _entry_points.start | default('', true) != '') %}
```

Comment:
Can we use a loop for action here, as for the governor?

`{% for __action, __action_config in _actions.items() %}`

Same comment about the condition.

Reply:
Not using a loop here because only start/stop/status actions relate to the resource provider. I'll work on the condition.